packages/synapse-react-client/jest.config.cjs

```diff
@@ -6,6 +6,7 @@ const esModules = [
   'mui-one-time-password-input',
   'p-limit',
   'yocto-queue',
+  '@ver0',
```

Comment:
`@react-hookz/web` upgrade pulls in some new transitive dependencies that are ESM only. Update the Jest config to properly handle these deps.